src/integrationTest/java/org/opensearch/security/DoNotFailOnForbiddenTests.java

```diff
@@ -48,15 +48,7 @@
 
 import static org.apache.http.HttpStatus.SC_CREATED;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.aMapWithSize;
-import static org.hamcrest.Matchers.allOf;
-import static org.hamcrest.Matchers.arrayContainingInAnyOrder;
-import static org.hamcrest.Matchers.arrayWithSize;
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasKey;
-import static org.hamcrest.Matchers.not;
-import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.Matchers.*;
```

Comment:
I think we don't want to use wildcard imports.

Reply:
I ran spotless apply on it and it didn't change anything, should I change it back?

Reply:
Should I create a follow up issue that fails the spotless apply if there is a * import?

Reply:
Reverted it here: [75877b7](https://github.com/opensearch-project/security/pull/4436/commits/75877b70a414fa7a5921042049ac94a4b49d3b83)